modules/account-lib/src/coin/avaxc/transaction.ts

```diff
@@ -0,0 +1,69 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
```

Comment:
I think we could re-use `eth`'s implementation of `Transaction`, like we did with [etc](https://github.com/BitGo/BitGoJS/blob/master/modules/account-lib/src/coin/etc/index.ts#L2)